.github/workflows/create_release_assets_cross.yml

```diff
@@ -54,14 +64,30 @@ jobs:
 
       - name: Rename Release
         run: |
-          mkdir assets
+          mkdir -p assets
           FILENAME=topgrade-${{github.event.release.tag_name}}-${{matrix.target}}
           mv target/${{matrix.target}}/release/topgrade assets
           cd assets
           tar --format=ustar -czf $FILENAME.tar.gz topgrade
           rm topgrade
           ls .
 
+      - name: Build Debian-based system package without autoupdate feature
+        run: |
+          rm -rf target/${{matrix.target}}
```

Comment:
Would like to document the reason why we remove the built binaries as well:

> "Remove the binary built by previous steps because we don't want the auto-update feature"

Reply:
Yep, makes sense.